#### Test 58751238ee11130_thali03_motorola-XT1039 Logs


```
--------- beginning of /dev/log/main
I/MDMCTBK (  275): NetlinkHandler, power_supply subsys
I/MDMCTBK (  275): NetlinkHandler, usbChargerStateChanged
I/MDMCTBK (  275): checkDefaultInst, current pid is = 275
I/MDMCTBK (  275): checkDefaultInst, pid match
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = /sys/class/power_supply/usb/online
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = 
E/MDMCTBK (  275): MdmCutbackHndler,Could not open ''
,D/AndroidRuntime( 3419): 
D/AndroidRuntime( 3419): >>>>>> AndroidRuntime START com.android.internal.os.RuntimeInit <<<<<<
D/AndroidRuntime( 3419): CheckJNI is OFF
D/dalvikvm( 3419): Trying to load lib libjavacore.so 0x0
D/dalvikvm( 3419): Added shared lib libjavacore.so 0x0
D/dalvikvm( 3419): Trying to load lib libnativehelper.so 0x0
D/dalvikvm( 3419): Added shared lib libnativehelper.so 0x0
D/dalvikvm( 3419): No JNI_OnLoad found in libnativehelper.so 0x0, skipping init
D/dalvikvm( 3419): Note: class Landroid/app/ActivityManagerNative; has 181 unimplemented (abstract) methods
I/MDMCTBK (  275): NetlinkHandler, power_supply subsys
I/MDMCTBK (  275): NetlinkHandler, usbChargerStateChanged
I/MDMCTBK (  275): checkDefaultInst, current pid is = 275
I/MDMCTBK (  275): checkDefaultInst, pid match
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = /sys/class/power_supply/usb/online
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = 
E/MDMCTBK (  275): MdmCutbackHndler,Could not open ''
I/MDMCTBK (  275): NetlinkHandler, power_supply subsys
I/MDMCTBK (  275): NetlinkHandler, usbChargerStateChanged
I/MDMCTBK (  275): checkDefaultInst, current pid is = 275
I/MDMCTBK (  275): checkDefaultInst, pid match
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = /sys/class/power_supply/usb/online
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = 
E/MDMCTBK (  275): MdmCutbackHndler,Could not open ''
E/memtrack( 3419): Couldn't load memtrack module (No such file or directory)
E/android.os.Debug( 3419): failed to load memtrack module: -2
D/AndroidRuntime( 3419): Calling main entry com.android.commands.am.Am
I/MDMCTBK (  275): NetlinkHandler, power_supply subsys
I/MDMCTBK (  275): NetlinkHandler, usbChargerStateChanged
I/MDMCTBK (  275): checkDefaultInst, current pid is = 275
I/MDMCTBK (  275): checkDefaultInst, pid match
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = /sys/class/power_supply/usb/online
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = 
E/MDMCTBK (  275): MdmCutbackHndler,Could not open ''
--------- beginning of /dev/log/system
I/ActivityManager( 1023): START u0 {flg=0x10000000 cmp=com.test.thalitest/.MainActivity} from pid 3419
D/AndroidRuntime( 3419): Shutting down VM
D/dalvikvm( 3419): GC_CONCURRENT freed 97K, 15% free 602K/704K, paused 0ms+0ms, total 2ms
,V/AlarmManager( 1023): sending alarm Alarm{41fed230 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{428d3900 type 2 com.google.android.gms}
,D/ConnectivityService( 1023): handleInetConditionChange: no active default network - ignore
,I/ClearcutLoggerApiImpl( 2042): disconnect managed GoogleApiClient
,V/AlarmManager( 1023): sending alarm Alarm{42828950 type 2 com.google.android.gms}
,E/global frequency( 1571): no tags to log
,D/Checkin ( 1571): publish the event [tag = MOT_CHECKIN event name = LOG]
I/BSUtils ( 1571): failed to get sys.build.version.checksum. Setting swChkSum to default: FFFF
I/BSUtils ( 1571): failed to get sys.build.version.checksum. Setting swChkSum to default: FFFF
I/global frequency( 1571): BcsDSCheckin.logProperties: root status from persist.qe isqe 0/0
D/Checkin ( 1571): publish the event [tag = MOT_CHECKIN event name = LOG]
D/Checkin ( 1571): publish the event [tag = DEV_ATTRIBS event name = SW]
,I/global frequency( 1571): BcsDSCheckin.events found events 2000
,D/Checkin ( 1571): publish the event [tag = MOT_CHECKIN event name = LOG]
,D/dalvikvm( 1571): GC_CONCURRENT freed 5474K, 32% free 11715K/17224K, paused 2ms+6ms, total 61ms
,D/dalvikvm( 1571): WAIT_FOR_CONCURRENT_GC blocked 30ms
,I/BSUtils ( 1571): failed to get sys.build.version.checksum. Setting swChkSum to default: FFFF
,D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(v1/cs/checkin)
D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: 
I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: 
D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
D/CCE     ( 1571): AppWSProxy - handleResponse(): error=RadioDownError errorText= statusCode=0 reqId=
,D/CCE     ( 1571): AppWSProxy - sendAIDLWSResponse() sending reponse
I/ErrorTranslator( 1571): unknown error code mapping for: 0
I/global frequency( 1571): BcsCore.status() called with error code=ERROR_FAIL
,D/Checkin ( 1571): publish the event [tag = MOT_CHECKIN event name = LOG]
I/global frequency( 1571): BcsTimer.onReceive checkin completed (0:ERROR_FAIL)
,D/Checkin ( 1571): publish the event [tag = MOT_CHECKIN event name = LOG]
,I/MDMCTBK (  275): NetlinkHandler, power_supply subsys
I/MDMCTBK (  275): NetlinkHandler, usbChargerStateChanged
I/MDMCTBK (  275): checkDefaultInst, current pid is = 275
I/MDMCTBK (  275): checkDefaultInst, pid match
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = /sys/class/power_supply/usb/online
I/MDMCTBK (  275): MdmCutbackHndler,readFromFile = 
,E/MDMCTBK (  275): MdmCutbackHndler,Could not open ''
,I/MMApiBackOffService( 1571): MMAPIWebServiceRequest backOff fired!
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,D/MMApiWebService( 1571): MMApiBackOffService told us it's okay to retry the waiting requests: 3
,D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(/v1/dp/devices.json)
,D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(v1/ns/list/messages)
,D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: devices.json
D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: 
E/MMApiProvisionService( 1571): ProvisionWS.Response: Missing mandatory message data in response from DM server...
E/MMApiProvisionService( 1571): ProvisionWS.Response.setError: error RadioDownError
I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: 
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiBackOffService( 1571): connectStatus(): app: MMAPIWebServiceRequest backing off: 300000 ms until next web service attempt
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiWebService( 1571): MMApiWebService told us not to continue at the moment with this request: 
I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: devices.json
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiBackOffService( 1571): connectStatus(): app: MMAPIWebServiceRequest already backing off for interval: 300000
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiWebService( 1571): MMApiWebService told us not to continue at the moment with this request: devices.json
D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(v1/us/devices/check)
D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: 
I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: 
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiBackOffService( 1571): connectStatus(): app: MMAPIWebServiceRequest already backing off for interval: 300000
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
I/MMApiWebService( 1571): MMApiWebService told us not to continue at the moment with this request: 
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,V/AlarmManager( 1023): sending alarm Alarm{42791d78 type 2 com.google.android.gms}
,I/VacuumService( 1215): Vacuum at: now=1455027776164 tag=VacuumService
,V/AlarmManager( 1023): sending alarm Alarm{428ab298 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{428ab370 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{427aca20 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{427b26f8 type 2 com.google.android.gms}
,D/ConnectivityService( 1023): handleInetConditionChange: no active default network - ignore
,W/SocketClient(  273): write error (Broken pipe)
,V/AlarmManager( 1023): sending alarm Alarm{4276dfd0 type 3 android}
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/AlarmManager( 1023): sending alarm Alarm{4204a7f0 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{42788778 type 3 android}
,I/MMApiBackOffService( 1571): MMAPIWebServiceRequest backOff fired!
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,D/MMApiWebService( 1571): MMApiBackOffService told us it's okay to retry the waiting requests: 3
D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(/v1/dp/devices.json)
D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(v1/ns/list/messages)
,D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: devices.json
E/MMApiProvisionService( 1571): ProvisionWS.Response: Missing mandatory message data in response from DM server...
D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: 
E/MMApiProvisionService( 1571): ProvisionWS.Response.setError: error RadioDownError
I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: 
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiBackOffService( 1571): connectStatus(): app: MMAPIWebServiceRequest backing off: 600000 ms until next web service attempt
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
I/MMApiWebService( 1571): MMApiWebService told us not to continue at the moment with this request: 
I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: devices.json
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiBackOffService( 1571): connectStatus(): app: MMAPIWebServiceRequest already backing off for interval: 600000
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
I/MMApiWebService( 1571): MMApiWebService told us not to continue at the moment with this request: devices.json
,D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(v1/us/devices/check)
,D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: 
I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: 
D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiBackOffService( 1571): connectStatus(): app: MMAPIWebServiceRequest already backing off for interval: 600000
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiWebService( 1571): MMApiWebService told us not to continue at the moment with this request: 
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,V/AlarmManager( 1023): sending alarm Alarm{42785f88 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{429a9960 type 2 com.google.android.gms}
,D/ConnectivityService( 1023): handleInetConditionChange: no active default network - ignore
,W/SocketClient(  273): write error (Broken pipe)
,V/AlarmManager( 1023): sending alarm Alarm{4295b980 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{428dd500 type 3 android}
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/AlarmManager( 1023): sending alarm Alarm{4276f8e0 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{428152b0 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{427354f0 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{4280bb38 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{4282a2a8 type 3 android}
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/AlarmManager( 1023): sending alarm Alarm{428a8448 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{427b27d0 type 2 android}
,V/AlarmManager( 1023): sending alarm Alarm{428967e0 type 2 com.google.android.gms}
,V/AlarmManager( 1023): sending alarm Alarm{427f7e30 type 0 com.android.vending}
,D/ConnectivityService( 1023): Sampling interval elapsed, updating statistics ..
,D/Finsky  ( 3050): [1] DailyHygiene.onStartCommand: Beginning daily hygiene
,D/ConnectivityService( 1023): Done.
,D/ConnectivityService( 1023): Setting timer for 720seconds
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,D/ConnectivityService( 1023): handleInetConditionChange: no active default network - ignore
,W/Finsky  ( 3050): [1] DailyHygiene$1.onErrorResponse: Unable to preload experiments: com.android.volley.NoConnectionError: java.net.UnknownHostException: Unable to resolve host "android.clients.google.com": No address associated with hostname
W/SocketClient(  273): write error (Broken pipe)
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,W/Finsky  ( 3050): [1] LibraryUpdateListener.onErrorResponse: Library replication failed: com.android.volley.NoConnectionError: java.net.UnknownHostException: Unable to resolve host "android.clients.google.com": No address associated with hostname
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,W/Finsky  ( 3050): [1] UpdateChecker$1$2.onErrorResponse: Update check failed: com.android.volley.NoConnectionError: java.net.UnknownHostException: Unable to resolve host "android.clients.google.com": No address associated with hostname
,D/Finsky  ( 3050): [1] DailyHygiene.access$600: Logging device features
,V/AlarmManager( 1023): sending alarm Alarm{4294cd28 type 0 com.android.vending}
D/Finsky  ( 3050): [1] DailyHygiene.reschedule: Scheduling new run in 32 minutes (failures=2)
,D/WearableService( 1215): callingUid 10022, callindPid: 1215
,D/DeviceConnectionService( 1215): client connected with version: 8296000
,D/Finsky  ( 3050): [264] FlushLogsService.onHandleIntent: Flushing event logs for [UVAT3FEWOLBYQGmBHvKHAcB3G_U]
,D/Finsky  ( 3050): [1] VerifyInstalledPackagesReceiver.onReceive: Skipping verification because network inactive
,D/Finsky  ( 3050): [1] WearSupportService$8.onConnectionFailed: onConnectionFailed: ConnectionResult{statusCode=API_UNAVAILABLE, resolution=null, message=null}
,D/Finsky  ( 3050): [1] WearSupportService.access$100: Dropping command=hygiene due to Gms not connected
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/AlarmManager( 1023): sending alarm Alarm{428188a8 type 0 com.android.vending}
,D/Finsky  ( 3050): [249] AppStatesReplicator.handleContentSyncResponse: Completed 0 account content syncs with 0 successful.
,D/Finsky  ( 3050): [1] ContentSyncService$5.onFinished: Installation state replication succeeded.
,V/AlarmManager( 1023): sending alarm Alarm{4297fd48 type 3 android}
,I/MMApiBackOffService( 1571): MMAPIWebServiceRequest backOff fired!
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,D/MMApiWebService( 1571): MMApiBackOffService told us it's okay to retry the waiting requests: 3
,D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(/v1/dp/devices.json)
D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(v1/ns/list/messages)
,D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: devices.json
E/MMApiProvisionService( 1571): ProvisionWS.Response: Missing mandatory message data in response from DM server...
D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: 
E/MMApiProvisionService( 1571): ProvisionWS.Response.setError: error RadioDownError
,I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: devices.json
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
I/MMApiBackOffService( 1571): connectStatus(): app: MMAPIWebServiceRequest backing off: 1200000 ms until next web service attempt
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiWebService( 1571): MMApiWebService told us not to continue at the moment with this request: devices.json
I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: 
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiBackOffService( 1571): connectStatus(): app: MMAPIWebServiceRequest already backing off for interval: 600000
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiWebService( 1571): MMApiWebService told us not to continue at the moment with this request: 
,D/MMApiWebService( 1571): doRequest() with req : MMApiWSRequest(v1/us/devices/check)
D/MMApiWebService( 1571): doRequest(): polling manager says we're not connected, returning with an error: 
D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiWebService( 1571): _inspectMMApiResponse(): found an error from a web service response: RadioDownError msg: RadioDownError req: 
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,I/MMApiBackOffService( 1571): connectStatus(): app: MMAPIWebServiceRequest already backing off for interval: 600000
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
I/MMApiWebService( 1571): MMApiWebService told us not to continue at the moment with this request: 
,D/Checkin ( 1571): publish the event [tag = MOT_CCE event name = LOG]
,V/AlarmManager( 1023): sending alarm Alarm{429079c8 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{4288bc70 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{42912108 type 3 android}
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,V/GLSActivity( 2042): AuthDelegateWrapperCreated with selected intent: Intent { cmp=com.google.android.gms/.auth.DefaultAuthDelegateService }
,W/SocketClient(  273): write error (Broken pipe)
,V/AlarmManager( 1023): sending alarm Alarm{429135b8 type 3 android}
,V/AlarmManager( 1023): sending alarm Alarm{429720b0 type 3 android}
,TIME-OUT KILL (timeout was 1200000ms),D/AndroidRuntime( 3526): 
D/AndroidRuntime( 3526): >>>>>> AndroidRuntime START com.android.internal.os.RuntimeInit <<<<<<
D/AndroidRuntime( 3526): CheckJNI is OFF
D/dalvikvm( 3526): Trying to load lib libjavacore.so 0x0
D/dalvikvm( 3526): Added shared lib libjavacore.so 0x0
D/dalvikvm( 3526): Trying to load lib libnativehelper.so 0x0
D/dalvikvm( 3526): Added shared lib libnativehelper.so 0x0
D/dalvikvm( 3526): No JNI_OnLoad found in libnativehelper.so 0x0, skipping init
D/dalvikvm( 3526): Note: class Landroid/app/ActivityManagerNative; has 181 unimplemented (abstract) methods
E/memtrack( 3526): Couldn't load memtrack module (No such file or directory)
E/android.os.Debug( 3526): failed to load memtrack module: -2
D/AndroidRuntime( 3526): Calling main entry com.android.commands.pm.Pm
W/PackageManager( 1023): Package named 'com.test.thalitest' doesn't exist.
D/dalvikvm( 1023): GC_EXPLICIT freed 946K, 9% free 18386K/20184K, paused 3ms+7ms, total 88ms
D/AndroidRuntime( 3526): Shutting down VM
D/dalvikvm( 3526): GC_CONCURRENT freed 94K, 15% free 564K/660K, paused 0ms+0ms, total 2ms

```
